projects/plugins/crm/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+### Changed
+- OAuth Connection: OAuth Connection"s" - updated typo. [#34131]
```

Comment:
> I'm not 100% sure how exactly changelog should be added for CRM. I updated the existing CRM changelog, but let me know if I should do otherwise.

This is not the Jetpack way to introduce the changelog. We don't add it directly to the `CHANGELOG.md` file. We add changelogs using individual files, see this PR example below:

https://github.com/Automattic/jetpack/pull/34605/files#diff-cef089a0a979d8206bff13839ad6665be0ae89a3e6f42df7313e6ddde775ee03.

Normally when you push the branch it performs some checks to see if a changelog file was added, if not, it prompts you to add one on the fly. Also, you can create that file manually before pushing it or using the `jetpack changelogger add {changelog-log-file}` command. Then, when we release JPCRM, one of the steps merges all the changelog files into the `CHANGELOG.md` file and finally removes them.

Reply:
Indeed, remove from here and add an entry via the `changelogger add` tool.

Reply:
Thanks! Removed this one and added one via changelogger tool.